test/run-spec-wasm2c.py

```diff
@@ -234,7 +235,7 @@ def _MaybeWriteDummyModule(self):
             self.commands.insert(0, dummy_command)
 
     def _WriteFileAndLine(self, command):
-        self.out_file.write('// %s:%d\n' % (self.source_filename, command['line']))
+        self.out_file.write('#line {line:d} "{name:s}"\n'.format(name=self.source_filename, line=command['line']))
```

Comment:
Maybe we could use f-string here instead.

e.g. `f'#line {command["line:]} "{self.source_filename}"'`

Reply:
No, we still target Python 3.5 allegedly. But this did remind us to clean up the `RunWithArgsForError` stuff we added to `utils.py` now that it is no longer needed...

Reply:
sgtm